strip-path/stripPath.js

```diff
@@ -1,8 +1,13 @@
 
-function stripPath(...args) {
-  return args;
+const path = require('path');
+
+function stripPath(pathToStrip, stripArgument) {
+  const relPath = path.relative('./', pathToStrip);
+  if (typeof stripArgument !== 'string') {
```

Comment:
Should we throw an error if any if the type is not a string for `pathToStrip` or `stripArgument`.

Reply:
@sourabhmodi20051 

>  if any if

Is it `if any of` or what you wrote?